### ISSUE TITLE
Documentation of practice (education & knowledge sharing)

### ISSUE THREAD

**Issue by yeqbfgxjiq** [aragoncoop/temp#13]
This looks like documenting the Aragon Cooperative practices so that others can recreate the model.

If so, first we need a practical model to make decisions and manage funds.

Once that is in place, we can document it. The README of this repo is the absolute minimum to fulfill that requirement. It would also be easy to build a beautiful simple [Docusaurus](https://docusaurus.io) website to display information about the Aragon Cooperative. I've already built a demo of what this *might* look like with the [COOP DAO Docs](https://aragoncoop.github.io/coop-dao-docs/), but that was for a more advanced model. I think it makes sense to create a simplified website explaining this workflow and where the Cooperative is at right now, and then upgrade it if/when we're ready.

**Comment by yeqbfgxjiq** [comment]
Closing this because with our new governance model, and it's associated documentation, we have met the minimum viable scope for this Issue. 

As mentioned in the comment above we can do a lot better, but first we need to prioritize the open AGP-40 deliverables. Once we have fulfilled the minimum scope for AGP-40, then we can go back and improve our work. Doing so would involve creating a new work proposal Issue to be voted on by the group. As such, closing this Issue so that we can focus on other open Issues.

**Comment by yeqbfgxjiq** [comment]
Reopening this Issue, because I think it makes more sense to have the community vote on whether or not they feel like the Issue has been satisfied. The process of closing Issues was not officially documented in the workflow, so I'll go ahead and create a [Meta Proposal](https://github.com/aragoncoop/temp#how-do-we-modify-this-protocol) for that now. My bad lol 

Edit: addressing that loophole with a Meta Proposal [here](https://github.com/aragoncoop/temp/issues/16)

**Comment by yeqbfgxjiq** [comment]
# Work Proposal: Documentation of practice (education & knowledge sharing)

<br>

### The AGP-40 deliverable that you’d like to work on

Documentation of practice (education & knowledge sharing)

It's been brought to my attention by current members that a user friendly single source of truth for the Aragon Cooperative would greatly improve the user experience for current and future members. This could involve a website with a landing page that explains what the Cooperative is from a high level, documentation explaining our operating procedures, a contributors guide, and a blog where people can catch up on our latest activity. It could also be augmented with links to our socials, GitHub, and any other important info. 

<br>

### The steps you’re going to take to move us towards shipping that deliverable
### The amount of time you think that’s going to take

I have experience building these things for the [Coop DAO Docs](https://aragoncoop.github.io/coop-dao-docs/), [1Hive](https://1hive.org/), and [Molochasaurus](https://www.burrrata.ch/molochasaurus/). The framework used for these websites is the same framework that hack.aragon.org uses. It's simple, beautiful, and it works. Based on my past experience I estimate this will take 24-32 hours (3-4 working days). To break that down:
- 4hrs to set up the skeleton with general content
- 4hrs to update docs on the current operating procedures
- 8hrs to create a contributors guide
- 8hrs get feedback from the group and updating content/styling
- 4hrs snafus and uknown unknowns

<br>

Leaving this open for discussion, and if sentiment is positive, will submit for a vote :)

<br>

### EDIT
Moved this to #19 because this is more of a general thread.

**Comment by julianbrooks** [comment]
Hey @burrrata , thanks for engaging with this...
For me the general idea of this is very good, couple of things though.
1) I know you're a FLOSS kinda person - me too. Docasuarus being facebook is something I'm struggling to justify us tacitly/passively supporting.

2) Some of my prior issues with the coop site were in regard the texts. I'd be up for us collaborating on something like this - perhaps you look after the code and I could sort the writing (of course we sketch it out between us). In an even more ideal world it'd be great if another of our cooperative members could manage the design and layout aspect for something like this.

I would propose my input being around 1 day over a week.

**Comment by julianbrooks** [comment]
I have been planning on making a video Rinkeby Aragon DAO HOWTO for several weeks now, and have spoken with a couple of people who are interested in talking the 'newb' role with this.

I think this would take at least a day.

Happy to discuss more and flesh the idea out if there's some interest in exploring this further...

**Comment by AaronAbuUsama** [comment]
Love it, This is a great initiative 🚀. Personally, not having a portal of information was one of the biggest pain points for me.

> I know you're a FLOSS kinda person - me too. Docasuarus being facebook is something I'm struggling to justify us tacitly/passively supporting.

Afaik Docasuarus is FLOSS, I don't have an issue with it anymore than I do using android but ¯\_(ツ)_/¯

I'd also be up for contributing (a little) time towards this. I've outlined a blog post about my experience contributing to the coop, helping on some Issues and what I have learned. It won't be ready for about a week. But if there's anything I can do just reach out on keybase.

**Comment by yeqbfgxjiq** [comment]
First, [Docusaurus](https://docusaurus.io/) is open source. 
Second, [Docusaurus is the framework used by hack.aragon](https://github.com/aragon/hack/issues/141). If it's good enough for Aragon, it's good enough for the Aragon Cooperative.

**Comment by yeqbfgxjiq** [comment]
@pythonpete32 A blog post sounds awesome! Eventually we could incorporate content like that into the Aragon Cooperative blog (because the new website will have a feature for this). 

Right now I think it makes sense to discuss it in #5 as part of the [Aragon Cooperative Blog](https://github.com/aragoncoop/temp/issues/5) initiative. While this is not a strict AGP-40 deliverable, it could nicely fit into #15 [Further suitable community-led, signaled and proposed opportunities](https://github.com/aragoncoop/temp/issues/15). There's already been positive sentiment and discussion in that direction, so if that's something you want to discuss/develop in #5 and then put to a vote I would support it :)

**Comment by yeqbfgxjiq** [comment]
<h1 align='center'> ! </h1>

I messed up in my Work Proposal here. As per the temp workflow I should have created a new Issue with my Work Proposal, but instead I created a comment in this thread. My bad. I'll keep the thread going since there's already comments here, but for future reference Work Proposals should be in their own threads :)

**Comment by julianbrooks** [comment]
> Afaik Docasuarus is FLOSS, I don't have an issue with it anymore than I do using android but ¯_(ツ)_/¯

I'm not sure the two are comparable. Yes I agree there's some pragmatic recognition of _necessary evil_ :) (e.g. here being M$-owned) but re android you basically have 2 options (w. IOS). In terms of website frameworks there are bazillions (a lot:).

No reason we can't do better, and yes, highlight and raise the ethical standards if required...

**Comment by topocount** [comment]
Okay so we all know @burrrata loves Github, and he's presented very good reasons for using Docausaurus, which relies on Github. I love using it as well, but I want to raise the issue of "accessibility" here, since I believe it to be a valid concern, with the question: **Does a more accessible alternative exist for maintaining documentation that is, itself, accessible?**

Absent of coming up with something completely new, the preeminent options for documentation right now are either something github-based (such as [readthedocs.io](https://docs.readthedocs.io/en/stable/open-source-philosophy.html) or Docasaurus), or [MediaWiki](https://en.wikipedia.org/wiki/MediaWiki), the platform upon which wikipedia, and many other documentation sites are built. Initially I was going to say that a github-based documentation engine was the clear choice, since editing things the traditional way in MediaWiki is really terse and intimidating. However, I think the new [visual editor](https://en.wikipedia.org/wiki/Wikipedia:VisualEditor) they have now is awesome! It's so good, I want to suggest using mediawiki instead, since the visual editor is much more accessible and abstracts away the need to learn git, or the esoteric nomenclature of Github

**Comment by yeqbfgxjiq** [comment]
@topocount 
FRT editing documentation and/or main page text in Docusaurus is done via markdown. The skeleton for the website/framework is setup via React, but after that all the content is filled in via vanilla Markdown docs. Super easy for people to edit and submit PRs.  

Also, people have stated that they want more than just docs. They want a single source of truth for user onboarding to the Cooperative. That means a homepage with logos and high level descriptions of the organization, social links, a blog for ongoing updates, user guides for the DAO apps, Cooperative bylaws, membership docs/guides. Docusaurus gives us all of that, and once the skeleton is built you can maintain it by editing Markdown documents. 

Furthermore, as part of rolling out a Docusaurus I would also roll out user guides on [how to use GitHub](https://github.com/aragon/hack/blob/master/docs-internal/github-guide.md) and a quickstart for editing stuff in Docusaurus.  

EDIT:
@topocount
Question: are you volunteering to build out a MediaWiki landing page and documentation for the Cooperative? Because if so you could totally submit a Work Proposal for that. Then the community could vote on which project they want to see built, or both. 
As is, this thread is getting very off topic very fast. I think that's because it's a general Issue for the AGP-40 deliverable and I accidentally posted my Work Proposal here. That's my bad. I'll move my Work Proposal out of this repo so that this can be a more general discussion thread :)

**Comment by AaronAbuUsama** [comment]
Markdown is super easy but there is a technical barrier to using github. That being said we're not talking about massive hurdles here. Also, I thinking about the people we are targeting would bring a bit more perspective to this discussion. 

Before you even talk about GitHub, the contributor has to know about public private key pairs, what a blockchain is etc. Yes, we're targeting non-devs but we're still talking about people with some technical skills.

So yes, the easier the better but also be mindful if our actual target contributors

**Comment by yeqbfgxjiq** [comment]
Aragon is a blockchain based project that lives on Ethereum. If people want to get involved in the Aragon community and learn to use DAOs, they're going to need to learn about how Ethereum works, how Aragon works, how DAOs work, and how the Aragon Cooperative works. You can't contribute to a project that has onchain governance while simultaneously avoiding things like blockchains, crypto, and formal governance protocols.

If people want to discuss general ideas, the [Aragon Forum](https://forum.aragon.org) is a great place for that. If people want to get more involved to start using DAOs and building education materials, the Aragon Cooperative is a great place for that. Being involved and actually building/doing things requires a higher level of sophistication than just speculating from the sidelines and having general discussions. People who want to **do things** and **build things** can do that here. People who just want to talk can go to the forum.

Also, if people are new and do not yet understand the basics of blockchains, Ethereum, and Aragon the Aragon Cooperative can serve as a great resource to help them learn. The key word here is "learn." You cannot maintain your current state of understanding while engaging in a new industry or community. If you want to get involved and learn, we can help. If you want to join a cooperative, but would rather stay away from crypto, formal governance, or on-chain voting this group is not for you.

**Comment by calcadarafael** [comment]
How does this last post work against the MVM? I cant see how we can be open to external collaboration with this cathedratic position on education.

> The Aragon Coop Manifesto
We are a community that is fundamentally engaged with the Aragon Project, and are composed of individuals that decided to come together to cooperatively and collectively voice our visions of the future of the Aragon ecosystem.
We stand for collaboration as a prominent tool to reach true decentralization 1 while generating genuine value within the Aragon Network and other communities at large.
We aim for fairness among ANT holders, the AA, developers, community members and users, within every interaction and/or fund allocation that could take place among them, we want to be defenders of the Aragon Manifesto and its Code of Conduct.
We strive to experiment with governance models and new tools to pave the pathway toward true freedom in this world. We will start with democratic one person, one vote mechanism, but will also try other suitable mechanisms such as ones based on merit, reputation, delegation, prediction markets, quadratic voting, etc. – adapting and evolving our own governance through time.
In a nutshell, the Aragon Cooperative aims to contribute not only to the Aragon Network with a grassroots approach but also to other ecosystems that want to embrace decentralization and autonomy within their organizational structures.

**Comment by LouisGrx** [comment]
Think we can have a single source of truth provided the content put up there is approved by the coop

**Comment by calcadarafael** [comment]
im one of the proponents of having a single source of truth to facilitate the communication inside the coop and how we/future learners will be onboarded to the platform.

AFAIK DAOs have a really flexible market fit as they can be used for soo much different things. That means we will onboard people with different technical skills. 

> " If people want to get involved in the Aragon community and learn to use DAOs, they're going to need to learn about how Ethereum works, how Aragon works, how DAOs work, and how the Aragon Cooperative works"

Having this kind of position is dogmatic and not really on par on what has been proposed by the manifesto and thats why its quoted there. 

>  If you want to get involved and learn, we can help. 

We also want something. We want this people using this platform. So why are we not open to the way those people do things?

This is what the manifesto is about. Again, education is a 2 way street. We learn how to communicate, they learn how to use the tools. There are even methods of teaching that say that we should learn how to teach it in their environment and not in ours. 

> If it's good enough for Aragon, it's good enough for the Aragon Cooperative.

Maybe not and thats could be one of the reason why  the Coop was created?

**Comment by AaronAbuUsama** [comment]
@calcadarafael
I don't see the big issue here. Clearly there is a minimal technical standard required to participate in a DAO. You need to know what ethereum is and how to manage your keys.  It's also clear we have a mandate to educate a "non-dev" audience. These will inevitably fall on differing ends of the technical spectrum. So we as a coop would aim to target that a range of users. I'm sure you would be great at producing materials and support for those people, your clearly passionate about it.

> AFAIK DAOs have a really flexible market fit as they can be used for soo much different things. That means we will onboard people with different technical skills.

To be honest, I am a little confused, what are you talking about. what your trying to on board these people onto? Aragon the platform or the Coop? If the coop then surely you have to know something to teach or at the bare minimum provide valuable signals. If you just look at the education function in isolation for a second, no education institution would onboard students into teaching and management positions before they had graduated right?

> im one of the proponents of having a single source of truth to facilitate the communication inside the coop and how we/future learners will be onboarded to the platform.

Then what's your proposal? Who is going to build it? How long is it going to take? @julianbrooks dosn't like it because of the FB connection. I respect that and if someone wants to do it on mediawiki or somewhere else, put a proposal in, I'll support that.  but really guys, come on, It just seems to me like people want to argue for the sake of it.

**Comment by calcadarafael** [comment]
@pythonpete32 
my issue is on taking an approach that is clearly dogmatic. Consciously or not. And specially when those go against the outline in the MVM.

Aragon is a manifesto based organisation that is being technological enabled, but that doesnt require that we dont hear or collaborate with non-tech/crypto people and thats why we should be aware of the design choices we make from early one in order to dont perpetuate our bias and exclude those voices. Its also on the manifesto.

ex:Someone that worked with law or with working groups their entire life can be as beneficial to the Coop as someone who siloed write code for the CLI. We should make sure both of them have a voice and a vote. 

There is already a barrier to entry and get a vote and we are now creating bigger ones by specifying things without first consulting with the group or understanding the target we are working on and the motivation for that. Im pointing those out when I see them and and pointing suggestion to improve it.
